pkg/execution/runner/runner.go

```diff
@@ -310,13 +311,67 @@ func (s *svc) handleMessage(ctx context.Context, m pubsub.Message) error {
 	return errs
 }
 
+// FindInvokedFunction is a helper method which loads all available functions, checks
+// the incoming event and returns the function to be invoked via the RPC invoke event,
+// or nil if a function is not being invoked.
+func FindInvokedFunction(ctx context.Context, evt event.Event, fl cqrs.ExecutionLoader) (*inngest.Function, error) {
+	if evt.Name != consts.InvokeEventName {
+		return nil, nil
+	}
+
+	fns, err := fl.Functions(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	name := evt.Data[consts.InvokeSlugKey]
```

Comment:
What's the reason for putting the slug key in `event.data` instead of a new top-level property? I don't think that's a blocker for this PR, but it feels icky to pollute the event data. Maybe that causes problems with some data validation libraries that don't allow excess properties? Or maybe that complicates event replay (when we add that feature)?

Reply:
We don't want to add top-level event properties for specific features.  The format we have is small and concise;  changing it impacts internals deeply.  If anything, we should add a single `ctx` map to the event data which allows for baggage and other event metadata to be included maybe.  This can be a future change.

One of the key aspects is that Inngest controls the creation of this event.  We can adjust this at any time without having to worry about SDK rollouts.